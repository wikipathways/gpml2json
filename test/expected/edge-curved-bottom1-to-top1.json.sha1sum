2ad5b2628398488b1a940669c212df1d68dfdcd4 ?-
