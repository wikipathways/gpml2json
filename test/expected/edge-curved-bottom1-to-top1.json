{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["e3b17","d5cad","cf964"],"drawAs":"rect","gpmlElementName":"Pathway","height":783.1967827425314,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":1045.87076968672,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20130813","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/a3a5b9bd7c1ab34450d15cf2a47dc26ab8960928bd68db4f95d20ebcd4d4ab6f"},"entityMap":{"d5cad":{"id":"d5cad","textContent":"GeneProduct","wpType":"GeneProduct","x":160.68118567178323,"y":115.33238139805435,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":33005,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"cf964":{"id":"cf964","textContent":"0021","wpType":"GeneProduct","x":12.46064141579781,"y":56.501359872350776,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":33006,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"e3b17":{"id":"e3b17","drawAs":"CurvedLine","borderWidth":1,"zIndex":33004,"points":[{"isAttachedTo":"cf964","attachmentDisplay":{"position":[0.25,1],"offset":[0,0]},"x":32.46064141579781,"y":76.50135987235078,"orientation":[0,1]},{"x":116.57091354379052,"y":96.50135987235078},{"isAttachedTo":"d5cad","attachmentDisplay":{"position":[0.5,0],"offset":[0,0]},"x":200.68118567178323,"y":115.33238139805434,"orientation":[0,1]}],"color":"#000000","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerStart":"TBar","markerEnd":"Arrow","isAttachedTo":["cf964","d5cad"]}}}
